Primo.js

```diff
@@ -363,37 +358,184 @@ function importPNX(text) {
 	item.callNumber = ZU.xpathText(doc, '//enrichment/classificationlcc');
 	
 	item.complete();
+}
+
+function cleanAuthor(str) {
```

Comment:
Can we rename that function because there is a function `ZU.cleanAuthor()` with the same name in the Zotero utilities? Maybe just `clean`...